Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml

```diff
@@ -1565,3 +1565,26 @@
     seedId: bungo
   - type: SpaceGarbage
   - type: BadFood
+
+- type: entity
```

Comment:
Please double check your [YAML conventions](https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#yaml-conventions)

Reply:
I have changed the order to be parent, id, name and then description.